js/main.js

```diff
@@ -1,2 +1,44 @@
 /* global data */
 /* exported data */
+
+var urlInput = document.querySelector('.photoUrl');
+var imgSrc = document.querySelector('.placeholder-img');
+urlInput.addEventListener('input', function (event) {
+
+  var imgUrl = event.target.value;
+  imgSrc.setAttribute('src', imgUrl);
+
+});
+
+var submitButton = document.getElementById('codeform');
+var entriesParsed = {};
+var previousFormEntries = localStorage.getItem('entry-form');
+
+if (previousFormEntries !== null) {
+  entriesParsed = JSON.parse(previousFormEntries);
+}
+
+if (previousFormEntries === null) {
+  localStorage.setItem('entry-form', JSON.stringify(data));
+}
+
+submitButton.addEventListener('submit', function (event) {
```

Comment:
You should make your event handler a named function rather than an anonymous function